pages/common/mysql.md

```diff
@@ -20,4 +20,4 @@
 
 - Execute SQL statements in a script file (batch file):
 
-`mysql {{database_name}} < {{script.sql}}`
+`mysql -h {{database_host}} -u {{user}} --password {{database_name}} -e "source {{/path/to/backup-file.sql]}" {{database_name}}`
```

Comment:
Perhaps you meant to remove the {{database_name}} after password ? Since I see you have added it to the end.

Also remove the ] after `.sql`. And the / before path.